lusidtools/general_utils/quick_func.py

```diff
@@ -0,0 +1,302 @@
+# This file contains a set of data manipulation functions
```

Comment:
not a big fan of `quick_func.py` as a name, I'd generally prefer something more descriptive such as `data_loading_funcs.py`